components/eventTypeAvailTimes.jsx

```diff
@@ -22,14 +23,30 @@ export default () => {
     setEventType(result.eventType);
   };
 
-  const fetchEventTypeSlotsData = async (params) => {
-    const result = await fetch(`/api/event_type_available_times${params}`).then(
-      (res) => res.json()
-    );
+  const fetchEventTypeSlotsData = async () => {
+    const result = await fetch(
+      `/api/event_type_available_times${queryParams}`
+    ).then((res) => res.json());
+
+    if (result.availableSlots.length === 0) {
+      //This takes care of if a user sets a certain date range when an event type can be scheduled, or how far in advance the event type can be scheduled.
+      let start = new Date(queryParams.split('=')[1].substring(0, 24))
```

Comment:
These lets could be const since we're not reassigning and just using for displaying data

Reply:
Take a look throughout the files and see which lets you can make a const instead 👍

Reply:
I was thinking the same, but

> Oh React how I've missed you so. Left a few comments

Thank you so much for reviewing this.

Reply:
I was thinking the same. I tend to use "const" but, since this is dynamic, essentially, I was thinking to use "let" as a best-practice/obsessive thing. Not sure if that's the way to go. I can go with "const".